classes/py1.html

```diff
@@ -0,0 +1,135 @@
+---
+layout: default
+title: "Intro to Programming with Python 1"
+permalink: /classes/py1
+---
+
+<!-- todo: custom layout? -->
+<div class="py1-wrapper">
+  <div class="py1-sidebar">
+    <p>how to get started in<br/>
+      <span class="py1-fancy-title">python</span>
+    </p>
+    <div class="py1-illustration">
+      <!-- <img alt="a filler murray" src="https://www.fillmurray.com/900/600"/> -->
+    </div>
+    <p>
+      confused about how to get started with 
+      python? here's a <strong>step-by-step learning plan</strong> 
+      that walks you through how to learn to make 
+      awesome programs, with no prior experience!
+    </p>
+    <hr class="py1-divider-sidebar"/>
+    <div class="py1-learned-skills-wrapper">
+      <p>
+	learned skills:
+      </p>
+      <!-- this part could be automated with markdown + loop -->
+      <ul class="py1-list py1-learned-skills-list">
+	<li><span>data types and operators</span></li>
+	<li><span>booleans and conditional statements</span></li>
+	<li><span>while loops and for loops</span></li>
+	<li><span>lists</span></li>
+	<li><span>creating functions</span></li>
+      </ul>
+    </div>
+  </div>
+  <!-- needs to materialize as a vertical line on desktop -->
+  <!-- and a horizontal line on mobile? -->
+  <!-- or implement using margin -->
+  <!-- <hr class="py1-divider"/> -->
+  <div class="py1-divider">
+  </div>
+  <div class="py1-curriculum">
+    <!-- types of boxes: -->
+    <!-- section (lessons and assignment), -->
+    <!-- lesson (card with outline, buttons and time estimate), -->
+    <!-- assignment (with button and time estimate), -->
+    <!-- project (layout slightly differs from assignment), -->
+    <!-- illustrations -->
+
+    <!-- generation of boxes could be automated with markdown + loop -->
+    <section class="py1-section" id="fundamentals">
+      <div class="py1-section-header">
+	<h1>fundamentals</h1>
+	<p>goals: understand what Python is and the building blocks to
+	  creating your first working program!</p>
+      </div>
+      <div class="py1-lessons-wrapper">
+	<div class="py1-lesson">
+	  <div class="py1-lesson-title-bar">
+	    <h2>getting started</h2>
+	    <p class="py1-step-number">#1</p>
+	  </div>
+	  <div class="py1-lesson-body">
+	    <ul class="py1-list py1-lesson-goal-list">
+	      <li class="py1-lesson-goal"><span>what is Python?</span></li>
+	      <li class="py1-lesson-goal"><span>why is it important?</span></li>
+	      <li class="py1-lesson-goal"><span>how is it used?</span></li>
+	    </ul>
+	  </div>
+	  <div class="py1-lesson-action-bar">
+	    <ul class="py1-list py1-lesson-action-list">
+	      <li class="py1-lesson-action"><a href="#">slides</a></li>
+	      <li class="py1-lesson-action"><a href="#">video</a></li>
+	      <li class="py1-lesson-action"><a href="#">worksheet</a></li>
+	    </ul>
+	    <p class="py1-lesson-duration">10 min</p>
```

Comment:
If the viewport gets too small, this stacks and overflows into the action bar - not entirely sure why, but probably something to do with how the flex wrapping is set up?